Formula/lib/libheinz.rb

```diff
@@ -0,0 +1,34 @@
+class Libheinz < Formula
+  desc "C++ base library of Heinz Maier-Leibnitz Zentrum"
+  homepage "https://jugit.fz-juelich.de/mlz/libheinz"
+  url "https://jugit.fz-juelich.de/mlz/libheinz/-/archive/v2.0.0/libheinz-v2.0.0.tar.bz2"
+  sha256 "e1e67da3997de1c28e60a9c5c7a45e383526b0fa18aefbb6e7c7d671b487da8a"
+  license "GPL-3.0-or-later"
```

Comment:
Doesn't seem right: https://jugit.fz-juelich.de/mlz/libheinz/-/blob/main/LICENSE?ref_type=heads is the BSD 0-clause license. Also see source headers.

Reply:
make sense, let me update it.